.azure/default-build.yml

```diff
@@ -22,4 +24,26 @@ jobs:
   
   - ${{ parameters.beforeBuild }}
   
+  - task: CMake@1
+    inputs:
+      cmakeArgs: .. ${{ parameters.cMakeRunArgs }} -DCMAKE_TOOLCHAIN_FILE=../submodules/vcpkg/scripts/buildsystems/vcpkg.cmake -DENABLE_TESTING=1 -DCMAKE_BUILD_TYPE=${{ parameters.configuration }}
+      workingDirectory: 'build.${{ parameters.configuration }}'
+    displayName: Create build files
+  - task: CMake@1
+    inputs:
+      cmakeArgs: --build . --config ${{ parameters.configuration }}
+      workingDirectory: 'build.${{ parameters.configuration }}'
+    displayName: Build client
+  - bash: "./build.${{ parameters.configuration }}/bin/signalrclienttests"
+    condition: eq( variables['Agent.OS'], 'Darwin' )
+    displayName: Run tests
+  - bash: "./build.${{ parameters.configuration }}/bin/signalrclienttests"
+    condition: eq( variables['Agent.OS'], 'Linux' )
+    displayName: Run tests
+    continueOnError: true
+    timeoutInMinutes: 2
+  - powershell: "& ./build.${{ parameters.configuration }}/bin/${{ parameters.configuration }}/signalrclienttests.exe"
```

Comment:
can we try adding `--gtest_output=xml:.\build.${{parameters.configuration}}\signalrclienttests.xunit.xml` (or similar) and publish that using a publish test results task? GTest should support outputting an xUnit-style report this way (according to some brief Googling). If it's not as simple as that, we can defer it to later, but I'd like to prioritize the whole CI flow working as we do in other repos.

Reply:
You mind if we do that in a separate PR? We have an issue tracking it.

Reply:
I don't entirely mind. It just seems like it's pretty simple to add now :). As long as it happens soon I'm OK with whatever.